tools/accuracy_checker/accuracy_checker/annotation_converters/criteo_kaggle_dac.py

```diff
@@ -83,6 +90,47 @@ def parse_sparse_features(self):
                 else:
                     ConfigError('Invalid configuration option {}'.format(feat))
 
+    def load_data_file(self):
+        if self.binary:
+            bytes_per_feature = 4
+            tar_fea = 1  # single target
+            den_fea = 13  # 13 dense  features
+            spa_fea = 26  # 26 sparse features
+            tad_fea = tar_fea + den_fea
+            tot_fea = tad_fea + spa_fea
+            self._bytes_per_entry = bytes_per_feature * tot_fea * self.batch
+            self._fea_shape = (self.batch, tot_fea)
+
+            self.count = math.ceil(os.path.getsize(self.src) / self._bytes_per_entry)
+            self.cat_feat = spa_fea
+
+            self.binfile = open(self.src, 'rb') if self.save_preprocessed_features else None
+        else:
+            data = np.load(self.src)
+            self._x_int = data['X_int']
+            self._x_cat = data['X_cat']
+            self._y = data['y']
+            self.count, self.cat_feat = self._x_cat.shape
+            self.count = math.ceil(self.count / self.batch)
```

Comment:
you can avoid import math if it is need only in this place, the same operation exists in numpy

Reply:
Fixed